src/app/fdctl/run/tiles/quic.c

```diff
@@ -102,12 +109,15 @@ run( fd_tile_args_t * args ) {
 }
 
 static long allow_syscalls[] = {
-  __NR_write,     /* logging */
-  __NR_fsync,     /* logging, WARNING and above fsync immediately */
-  __NR_getpid,    /* OpenSSL RAND_bytes checks pid, temporarily used as part of quic_init to generate a certificate */
-  __NR_getrandom, /* OpenSSL RAND_bytes reads getrandom, temporarily used as part of quic_init to generate a certificate */
-  __NR_madvise,   /* OpenSSL SSL_do_handshake() uses an arena which eventually calls _rjem_je_pages_purge_forced */
-  __NR_mmap,      /* OpenSSL again... deep inside SSL_provide_quic_data() some jemalloc code calls mmap */
+  __NR_write,       /* logging */
+  __NR_fsync,       /* logging, WARNING and above fsync immediately */
+  __NR_getpid,      /* OpenSSL RAND_bytes checks pid, temporarily used as part of quic_init to generate a certificate */
+  __NR_getrandom,   /* OpenSSL RAND_bytes reads getrandom, temporarily used as part of quic_init to generate a certificate */
+  __NR_madvise,     /* OpenSSL SSL_do_handshake() uses an arena which eventually calls _rjem_je_pages_purge_forced */
+  __NR_mmap,        /* OpenSSL again... deep inside SSL_provide_quic_data() some jemalloc code calls mmap */
+  __NR_sendto,      /* allows to make requests on netlink socket */
+  __NR_recvfrom,    /* allows to receive responses on netlink socket */
+  __NR_sched_yield, /* sched_yield used by log under certain circumstances */
```

Comment:
Do we need `sched_yield`? What is the call stack?

Reply:
No. We don't need it. If it gets called it's a mistake